gammapy/maps/wcsnd.py

```diff
@@ -589,6 +585,63 @@ def get_spectrum(self, region=None, func=np.nansum, weights=None):
 
         return self.to_region_nd_map(region=region, func=func, weights=weights)
 
+    def binary_erosion(self, margin, from_edges=False):
```

Comment:
Minor: we typically name the methods after the verb, like `.convolve()` (and not `.convolution()`). So my proposed name here would either be `.erode()` or `.binary_erode()`.